jvm/sargon-android/src/main/java/com/radixdlt/sargon/samples/ResourceAddressSample.kt

```diff
@@ -24,6 +26,10 @@ object ResourceAddressSampleMainnet: Sample<ResourceAddress> {
 
     override fun other(): ResourceAddress = candy
 
+    override fun random(): ResourceAddress = newResourceAddressSampleRandom(
```

Comment:
maybe it will be usefull with a `fun random(networkID: NetworkID)` too? And then `random` can call it with Mainnet as default?